### DISCUSSION TITLE
Full path to SSH config?

### DISCUSSION THREAD

**Discussion by czgregory** [git-lfs/git-lfs#4720] [Q&A] [answered]
I have got a question regarding SSH configuration and LFS.
In my environement I use the GIT_SSH_COMMAND var to set up communication with Git. When I give a full path to a SSH config file (/path/to/config) clone works.
But with a local path (./config) clone ends with en error:

Error downloding object path/to/file/in/repo: Smudge error: Error downloding path/to/file/in/repo: Batch request: Can't open user config file ./config: No such file or directory: exit status 255.

Have you got any idea why a full path is neccessary? Is it lfs or maybe SSH requirement?

Greg

**Comment by bk2204** [comment, marked as answer]
Both Git and Git LFS can change the directory when they start up, such that they end up in the root of the repository.  Neither one guarantees the directory in which they invoke the command in `GIT_SSH_COMMAND`, and as such, it isn't necessarily safe to assume that the directory you're using on the command line is the same one as the SSH command is invoked in.

**Comment by czgregory** [comment]
Hi 

Thank you for the answer. You are are right, I forgot that git can change a dir where it does its job.
